### PR TITLE
Add Appium and tests for it

### PR DESCRIPTION
**Pull request by APshenkin**
https://github.com/Codeception/CodeceptJS/issues/222

### CODE REVIEW

File: test/helper/Appium_test.js [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+let should = require('chai').should();
+let app;
+let assert = require('assert');
+let path = require('path');
+let fs = require('fs');
+let fileExists = require('../../lib/utils').fileExists;
+let AssertionFailedError = require('../../lib/assert/error');
+let formContents = require('../../lib/utils').test.submittedData(path.join(__dirname, '/../data/app/db'));
+let expectError = require('../../lib/utils').test.expectError;
+let webApiTests = require('./webapi');
```
Comment (DavertMik) [inline test/helper/Appium_test.js]: never used...

File: lib/helper/Appium.js [hunk 2]
```diff
@@ -0,0 +1,613 @@
...
+    return this.browser.isLocked().then(function (res) {
+      return truth('device', 'to be locked').negate(res.value);
+    });
+  }
+
+  /**
+   * Check the device orientation
+   * Appium: support Android and iOS
+   */
+  seeOrientationIs(orientation) {
```
Comment (DavertMik) [inline lib/helper/Appium.js]: what are possible options for `orientation`?
Code examples please ;)

File: lib/helper/Appium.js [hunk 3]
```diff
@@ -0,0 +1,613 @@
...
+    return this.browser.settings().then(function (res) {
+      return res.value;
+    });
+  }
+
+  /**
+   * Switch to the specified context
+   * Appium: support only Android
+   */
+  switchToContext(value) {
```
Comment (DavertMik) [inline lib/helper/Appium.js]: What is a context? Code examples please )

**Comment by ecampii** [comment]
can we have it through npm install?

**Comment by DavertMik** [comment]
@ecampii yes, once the new version released.
But we need to prepare documentation for the release and test everything nicely )

**Comment by ecampii** [comment]
thanks.

**Pull request merged**